.github/workflows/pull-request-workflow.yml

```diff
@@ -22,15 +22,10 @@ jobs:
       uses: ruby/setup-ruby@v1.81.0
       with:
         bundler-cache: true
-    - name: Run pod install
-      run: |
-        set -eo pipefail
-        export LIB_VERSION=$(git describe --tags `git rev-list --tags --max-count=1`)
-        bundle exec pod install --project-directory=Example
     - name: Run tests (JustTweak)
       run: bundle exec fastlane unit_tests_just_tweak device:'iPhone 11'
     - name: Run tests (TweakAccessorGenerator)
-      run: bundle exec fastlane unit_tests_tweak_accessor_generator
+      run: bundle exec fastlane unit_tests_tweak_accessor_generator device:'iPhone 11'
```

Comment:
I don't think you want this. This is a CLI tool that runs on mac, not on iPhone 👀

Reply:
yess going to remove it.